perf/pulsar-perf-go.go

```diff
@@ -69,6 +70,15 @@ func main() {
 		Use: "pulsar-perf-go",
 	}
 
+	go func() {
```

Comment:
In my opinion this should not be started by default and should be controlled with a cli flag. Also, if you run two process only one server will start.

Reply:
By default, `pprof` does not collect project metrics. So it will not cause any loss in performance. 

But if we do this, we can provide the user with a `pprof` which is directly accessed by the browser. What's wrong?

In this way, people who use the pulsar-go-client in the production environment can also monitor their running status through the browser at all times, instead of collecting each time through the `go tool pprof`.

> Also, if you run two processes only one server will start. Why is this being added?

Are you sure? When testing in my local environment, we can access it in the browser through `localhost: 8889`, or we can use `go tool pprof -http=:8081 $HOME/pprof/pprof.samples.cpu.004.pb.gz` to access.

Reply:
This feature is also raised by users using pulse-client-go. I think this feature is reasonable, we should provide this way, what do you think?

Reply:
I'm not against the feature i just think it should be hidden behind a cli flag and not started by default. There is already a FlagProfile and a profiling server is started when that is enabled. Can this profiling be added to that server? I'm confused on why two need to be started.

If you start `pulsar-perf produce` and `pulsar-perf consume` on your laptop won't the server try to bind to same port and one of them not start?